docker-compose.yml

```diff
@@ -390,6 +410,16 @@ services:
       PHP_OPCACHE_VALIDATE_TIMESTAMPS: 1
       PHP_IDE_CONFIG: serverName=lpa-admin-app
       OPG_LPA_COMMON_ADMIN_ACCOUNTS: "${OPG_LPA_COMMON_ADMIN_ACCOUNTS}"
+    healthcheck:
+      test:
```

Comment:
Is the container testing its own web endpoint here? Is this called by docker behind-the-scenes when other containers want to healthcheck this one?

Reply:
The other containers won't be aware of the health check; the command will run inside of the container that it is targeting. The command according to the below schedule:

      interval: 10s - How often the check is run
      timeout: 15s - If the command takes longer than this to execute, it's considered a failed command.
      retries: 3 - The number of times the command must fail (either timeout or non-zero exit code)
      start_period: 90s - For 90 seconds after the container starts, failed commands won't count towards the retry count unless one of the commands pass then subsequent commands will count towards the retries.
      
If a container is unhealthy (i.e. 3x failed requests) then it will be restarted by the Docker daemon.